apify-api/openapi/components/schemas/actor-pricing-info/PayPerEventActorPricingInfo.yaml

```diff
@@ -0,0 +1,18 @@
+title: PayPerEventActorPricingInfo
+allOf:
+  - $ref: "./CommonActorPricingInfo.yaml"
+  - type: object
+    required:
+      - pricingModel
+      - pricingPerEvent
+    properties:
+      pricingModel:
+        type: string
+        enum: [PAY_PER_EVENT]
+      pricingPerEvent:
+        type: object
+        additionalProperties:
+          $ref: ./ActorChargeEvent.yaml
```

Comment:
Q: I think this. might be still inaccurate. The `pricingPerEvent` is object such as this
```
{actorChargeEvents: {
    [EVENT_NAME]: {
        "eventTitle": string,
        "eventDescription": string,
        "eventPriceUsd": number
    }
}
```
I haven't defined OpenAPI spec before, but the spec doesn't seem to represent this kind of object exactly. It seems, that it fails to address the part where `pricingPerEvent` contains additional `actorChargeEvents` which is map from event name to the ActorChargeEvent itself.

Reply:
correct, nice catch

Reply:
Nice catch, fixed